gradle.properties

```diff
@@ -12,9 +12,9 @@ runGenerators = true
 
 pluginGroup = mobi.hsz.idea
 pluginName = .ignore
-pluginVersion = 3.2.3.201
-pluginSinceBuild = 201
-pluginUntilBuild = 202
+pluginVersion = 3.2.3.202
+pluginSinceBuild = 202
```

Comment:
I tried to build this locally and got an error. Because `ideaVersion` is `201.xx` the build complains that it's invalid due to the `pluginSinceBuild` value being `202`.

```
2020-07-28 13:50:29,508 [   1173]  ERROR - llij.ide.plugins.PluginManager - Problems found loading plugins:
  The .ignore (id=mobi.hsz.idea.gitignore, path=/idea-gitignore/build/idea-sandbox/plugins/idea-gitignore) plugin is incompatible (reason: since build 202.0 > IC-201.7846.76, target build range is 202 to 203)
java.lang.Throwable: Problems found loading plugins:
  The .ignore (id=mobi.hsz.idea.gitignore, path=/idea-gitignore/build/idea-sandbox/plugins/idea-gitignore) plugin is incompatible (reason: since build 202.0 > IC-201.7846.76, target build range is 202 to 203)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:146)
        at com.intellij.ide.plugins.PluginManagerCore.prepareLoadingPluginsErrorMessage(PluginManagerCore.java:706)
        at com.intellij.ide.plugins.PluginManagerCore.prepareLoadingPluginsErrorMessage(PluginManagerCore.java:1115)
        at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1670)
        at com.intellij.ide.plugins.PluginManagerCore.loadAndInitializePlugins(PluginManagerCore.java:1904)
        at com.intellij.ide.plugins.PluginManagerCore.lambda$initPlugins$6(PluginManagerCore.java:1310)
        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1705)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
2020-07-28 13:50:29,511 [   1176]  ERROR - llij.ide.plugins.PluginManager - IntelliJ IDEA 2020.1.2  Build #IC-201.7846.76
```